resources/js/components/SignIn.js

```diff
@@ -48,18 +97,31 @@ const SignIn = () => {
           <Typography component="h1" variant="h5">
             Sign in
           </Typography>
-          <Box component="form" noValidate sx={{ mt: 1 }}>
+          {
+            user_auth.errors === true ?
```

Comment:
no need to elaborate the whole condition if the variable is boolean.

`variable_that_returns_a_boolean ? <what_to_do_when_true> : <what_to_do_when_false>`

Reply:
Shortened the condition in the ternary operation and removed unnecessary conditions. This is resolved